### PR TITLE
remove exporter label at cleanup.sh

### PR DESCRIPTION
**Pull request by YanjieGao**

### CODE REVIEW

File: service-deployment/bootstrap/prometheus/cleanup.sh.template [hunk 1]
```diff
@@ -26,3 +26,12 @@ configmap/prometheus-configmap"
 for instance in ${INSTANCES}; do
   kubectl delete --ignore-not-found --now ${instance}
 done
+
+{% for host in machinelist %}
+    {% if 'prometheus' in machinelist[ host ] and machinelist[ host ][ 'prometheus' ] == 'true' %}
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/cleanup.sh.template]: I found your label of prometheus and node-exporter share the same key? So prometheus and node-exporter can't deployed to the same node.

Comment (YanjieGao) [reply service-deployment/bootstrap/prometheus/cleanup.sh.template]: refined

File: service-deployment/bootstrap/prometheus/node-label.sh.template [hunk 2]
```diff
@@ -22,6 +22,6 @@
 kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} prom=prom
     {% endif %}
     {% if 'node-exporter' in machinelist[ host ] and machinelist[ host ][ 'node-exporter' ] == 'true' %}
-kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} prom=node-exporter
+kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} node-exporter=node-exporter
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/node-label.sh.template]: I will be better to use the same style. Like node-exporter="true"

Comment (YanjieGao) [reply service-deployment/bootstrap/prometheus/node-label.sh.template]: changed

File: service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template [hunk 3]
```diff
@@ -29,7 +29,7 @@ spec:
       name: node-exporter
     spec:
       nodeSelector:
-        prom: node-exporter
+        node-exporter: node-exporter
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template]: Please change this to "true". And the same to prom's yaml

File: service-deployment/bootstrap/prometheus/prometheus-deployment.yaml.template [hunk 4]
```diff
@@ -30,7 +30,7 @@ spec:
       hostNetwork: true
       hostPID: true
       nodeSelector:
-        prom: prom
+        prom: true
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/prometheus-deployment.yaml.template]: Add quotation ""

File: service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template [hunk 5]
```diff
@@ -29,7 +29,7 @@ spec:
       name: node-exporter
     spec:
       nodeSelector:
-        prom: node-exporter
+        node-exporter: true
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template]: Add quotation ""

File: service-deployment/bootstrap/prometheus/node-label.sh.template [hunk 6]
```diff
@@ -19,9 +19,9 @@
 
 {% for host in machinelist %}
     {% if 'prometheus' in machinelist[ host ] and machinelist[ host ][ 'prometheus' ] == 'true' %}
-kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} prom=prom
+kubectl label nodes {{ machinelist[ host ][ 'nodename' ] }} prom="true"
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/node-label.sh.template]: Remove quotation in node-label.sh.template

**Pull request merged**